pkg/compare/compare_test.go

```diff
@@ -92,6 +92,10 @@ func checkFile(t *testing.T, fileName, value string) {
 	if err != nil {
 		t.Fatalf("test %s failed reading .golden file: %s", fileName, err)
 	}
+	fmt.Println(string(expected))
```

Comment:
i think its a little to verbose, why do you think its necessary?

Reply:
Oh sorry that was to help me when debugging. Easier to read the println than to look at the string